### PR TITLE
[Project Submission] CryptoChampz

### PR DESCRIPTION
**Pull request by cds95**

### CODE REVIEW

File: projects/cryptoChampz.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+We use IPFS to store the images for our NFTs.
+
+## Project Status
+
+Under development
+
+## Previews
+
+https://i.imgur.com/vwOKoIg.png
```
Comment (autonome) [inline projects/cryptoChampz.md:25]: Can you format so it shows up as an image?

File: projects/cryptoChampz.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+ipfs://QmSRp9STByPHrurTCvpwgBfz8ZX5FbkxamXjEk7HRw7PDr/Demon_B.png
```
Comment (autonome) [inline projects/cryptoChampz.md:27]: ditto

File: projects/cryptoChampz.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+NFT collectors and Blockchain game developers
+
+## Rough estimated user base (if applicable)
+
+Not yet launched.
+
+## Github repo
+
+https://github.com/cds95/crypto-champions
```
Comment (autonome) [inline projects/cryptoChampz.md:39]: Your source code must be licensed MIT open source license in order to qualify for the grant.

**Review by autonome — CHANGES_REQUESTED** [review]
Needs a few changes.

File: projects/cryptoChampz.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+5
+
+### Team members
+
+CultureVictory - Product Mgmt
+CDS95 - Full stack Web3 dev
+Oozyx - Blockchain dev
+C4RE B34R - Art
+Architect - Art
```
Comment (autonome) [inline projects/cryptoChampz.md:61]: Can you link each Github account please?

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/cryptoChampz.md (hunk 3 above)
Comment (cds95) [reply projects/cryptoChampz.md:37]: Updated repo to have MIT license.

File: projects/cryptoChampz.md (hunk 1 above)
Comment (cds95) [reply projects/cryptoChampz.md]: Formatted this as an image.

File: projects/cryptoChampz.md (hunk 2 above)
Comment (cds95) [reply projects/cryptoChampz.md]: I've temporarily removed this as the link isn't working.  I will add it back once we have it back up.

File: projects/cryptoChampz.md (hunk 4 above)
Comment (cds95) [reply projects/cryptoChampz.md]: Only cds95, Oozyx and CultureVictory have GitHub accounts.

**Comment by cds95** [comment]
@autonome Hi Dietrich, I apologize for the delayed response as I didn't see your comments.  I've made some changes but left a few questions.  Please feel free to reach out if you find any issues with the PR.

**Comment by autonome** [comment]
Looks good, thanks!

**Pull request merged**